dataflows-inline/car-processing/README.MD

```diff
@@ -162,7 +162,10 @@ The window is flushed to the `car-colors` topic every 30 seconds.
 Watermark tracks the cutoff timestamp for tumbling windows.
 
 ```bash
->> show state count-by-color/count-by-color/watermark
+show state count-by-color/count-by-color/watermark
+```
+
+```bash
  Key      Value                     Window
  default  2024-03-15T13:44:30.043Z  *
```

Comment:
I don't know if it matters but in a similar show state example in `dataflows-inline/split-sentence/README.MD` below you removed the output of the show state command from the README

Reply:
I figured it's too long and the display is repetitive. We can bring back if needed.